package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stripe",
-  "version": "4.8.0",
+  "version": "4.8.1",
```

Comment:
I think by semver we'd bump the minor version rather than patch.

Want to just leave this out of the pull though? I'll take care of it and the rest of the release separately.